src/validations.json

```diff
@@ -0,0 +1,11 @@
+{
+    "do_not_delete": [
+        "azurerm_resource_group.main",
```

Comment:
I'm loving this list. It looks good to me but we should put the bundle through its paces a few times in sandbox in case any params can cause a destroy and recreate w/ the resources that are newer to us like `azurerm_monitor_diagnostic_setting`